.changeset/flat-chicken-repeat.md

```diff
@@ -0,0 +1,7 @@
+---
+'changesets-gitlab': minor
+---
+
+Add missing dependencies to the package.json
```

Comment:
![image](https://user-images.githubusercontent.com/8336744/142770264-0906e8a4-44db-4d60-b3bc-c59d7ca48a3b.png)

Can you split them into different changesets?

Reply:
Done